### ISSUE TITLE
Cutting single video in track. Control+K?

### ISSUE THREAD

**Issue by gasparfm** [OpenShot/openshot-qt#451]
I noticed there are no scissors as it was on version 1.4 and I know I have the new split tool (but it's not very useful for me). I try to cut a single video inside a track. I can right click the video choose cut and then "keep both sides" and my job is done. But the shortcut I see on this option is Control+K. But when I press this keys all tracks are being cut and I don't want that, just want to cut current track.

**Comment by jreynojustin** [comment]
We have made some changes in our newest daily build. We reintroduced the scissors as a razor to cut video segments. Take a look at the newest build and let us know your thoughts!

**Comment by amaliepiers** [comment]
I just downloaded Openshot. I can't see how to deselect the razor tool. Once I have made a cut I can right click to remove a section, but it makes another cut as well as deleting. I have tried every button and menu I can find. I thought I could just move the clips I want to another track, but the move tool only works back or forth on the video. If I go to move a clip I'm back with the razor tool... Help?
thank you!
